server/services/overviewPageService.ts

```diff
@@ -55,7 +44,6 @@ export default class OverviewPageService {
     private readonly allocationManagerApiClientBuilder: RestClientBuilder<AllocationManagerClient>,
     private readonly keyworkerApiClientBuilder: RestClientBuilder<KeyWorkerClient>,
     private readonly incentivesApiClientBuilder: RestClientBuilder<IncentivesApiClient>,
```

Comment:
Assume this will be removed in next PR as no longer used

Reply:
Yeah that has come out. Will do a full tidy up at the end too